tests/neg-custom-args/fatal-warnings/i17613b/i17613b.scala

```diff
@@ -0,0 +1,33 @@
+// scalac: -Xlint:type-parameter-shadow
+
+object i17613b:
```

Comment:
Maybe also add tests where:
 - the shadowed val comes from one of the multiple inherited traits
 - Shadowing of types in type lambdas, match types, polymorphic function types
 - Shadowing vals in lambdas
 - There are two unnamed givens of the same type on different levels of nesting